### ISSUE TITLE
JWPLAYER videos not disabled

### ISSUE THREAD

**Issue by BillH99999** [Eloston/disable-html5-autoplay#141]
JWPLAYER videos are still playing on www.seattlepi.com.  Should they be disabled by the plugin?

Thanks,
Bill

**Comment by Eloston** [comment]
Yes, but the extension's not written very well and development has stagnated. No one has taken up development yet.

Chromium 57 introduced a feature to prevent autoplay and autobuffering **only** on videos originating from a foreign domain, which will block some advertising videos. Also, I believe nothing can autoplay in a background tab.

**Comment by BillH99999** [comment]
Thanks for the reply.  That is not what I was hoping to hear.  I hope someone starts to work on it again.  In the meantime do you know of any other HTML5 autoplay blockers that work?

Thanks,
Bill

**Comment by Eloston** [comment]
Sorry, I don't. But I'm hoping that Google will expand upon what they have in version 57, because they have more input (user inputs, for example) and lower-level controls that can do a better job than any extension can.

**Comment by BillH99999** [comment]
OK... thanks for the replies.

Bill

**Comment by Eloston** [comment]
We should leave this open since you provided another website to test autoplaying. Might be helpful at some point.

**Comment by kbdank71** [comment]
I've noticed the same issue on thv11.com with the jwplayer not being disabled.  The one I found was http://www.thv11.com/news/local/improvements-coming-for-busy-bowman-and-markham-intersection-in-little-rock/433019808 although I checked several news stories and they all had the same result.

Just figured I'd add a me too for the jwplayer problem.
